rke/templates/coredns.go

```diff
@@ -266,7 +269,11 @@ spec:
           - --target=Deployment/coredns
           # When cluster is using large nodes(with more cores), "coresPerReplica" should dominate.
           # If using small nodes, "nodesPerReplica" should dominate.
+{{if .LinearAutoscalerParams}}
```

Comment:
This version doesnt support `"preventSinglePointFailure":true`, will this break with the defaults set?

Reply:
Looks like it's unmarshaled into the linearParams struct, so the version of cluster-proportional-autoscaler that doesn't have `preventSinglePointFailure` will ignore that field during unmarshal